engine/proofview.mli

```diff
@@ -178,6 +178,9 @@ val tclBIND : 'a tactic -> ('a -> 'b tactic) -> 'b tactic
     it's a specialized "bind". *)
 val tclTHEN : unit tactic -> 'a tactic -> 'a tactic
 
+(** Maps a tactic over its result. *)
+val tclFMAP : ('a -> 'b) -> 'a tactic -> 'b tactic
```

Comment:
I just noticed that this already exists, it is called Proofview.Monad.map

Reply:
Removed in 8bd694a49b